### ISSUE TITLE
Ability to select where to stream

### ISSUE THREAD

**Issue by sukiletxe** [rzane/dlnanow#2]
We users could benefit in being able to choose the device to which we want to stream. This could be done by using `dlnacasts.players`, by adding a command line switch to list them and another to select which one to stream to, letting the user specify just the IP address.
Thanks for this great program! It's the most complete one of its kind in NPM.

**Comment by rzane** [comment]
Sounds like a great feature. It sounds like you've investigated this a little bit, so would you be willing to submit a PR?

I'd recommend https://github.com/SBoudrias/Inquirer.js/ for displaying the options.

**Comment by sukiletxe** [comment]
I don't think so. I don't know JS, and even though I may be able to learn enough of it to do this, I'm not the best person to test if my pr would work before submitting (see the last comment in #1).
